### PR TITLE
[no ticket] bump scalatest version

### PR DESCRIPTION
**Pull request by Qi77Qi**
* Bumped scalatest version to latest. 3.0.x to 3.1.x is breaking change, hence the majority of the diff. Used [this](https://github.com/scalatest/autofix/tree/master/3.1.x#autofix-for-scalatest-31x) for migrating.
* Remove Bloop.
* Remove newrelic from dependencies file.
* Bump google2 to latest version
* Bump liquibase version. I upgraded version in leo a while ago, and noticed no breaking change.

---

**PR checklist**

- [ ] I've followed [the instructions](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -6,8 +6,6 @@ addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.2.5")
 
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.1")
 
-addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.2")
```
Comment (Qi77Qi) [inline project/plugins.sbt:9]: sbt 1.4.4 comes with `dependencyTree` task

File: project/Settings.scala [hunk 2]
```diff
@@ -33,7 +33,7 @@ object Settings {
     "-language:existentials", // Existential types (besides wildcard types) can be written and inferred
     "-unchecked", // Enable additional warnings where generated code depends on assumptions.
     "-Xcheckinit", // Wrap field accessors to throw an exception on uninitialized access.
-    "-Xfatal-warnings", // Fail the compilation if there are any warnings.
+//    "-Xfatal-warnings", // Fail the compilation if there are any warnings.
```
Comment (dvoet) [inline project/Settings.scala]: did you mean to remove this?

Comment (Qi77Qi) [reply project/Settings.scala]: ah good catch..this was for running scalafix..will revert back

**Review by dvoet — APPROVED** [review]
thanks Qi, you're the best!

**Comment by Qi77Qi** [comment]
@dvoet  I'm actually going to try migrating sam to 2.13 in a followup PR..if it's easy, I'll just do it :) (selfishly I'd like to drop 2.12 in wb-libs :) )

**Review by marctalbott — APPROVED** [review]
Thanks for doing this!

**Comment by Qi77Qi** [comment]
fiab start failed with 

```
17:34:12 [ERROR] POST https://firecloud-orchestration-fiab.dsde-qa.broadinstitute.org:23443/register/profile failed with status 409
```

jenkins retest

File: automation/hotswap.sh [hunk 3]
```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# Updates the Leo jar of a FIAB to reflect current local code.
+# Run using "./automation/hotswap.sh <your FIAB name>" at the root of the sam repo clone
```
Comment (Qi77Qi) [inline automation/hotswap.sh:4]: handy script for swapping sam jar in fiab...copied over from leonardo

**Pull request merged**